src/nodes/corenodes/input/noise_image_node/noise_image_node.py

```diff
@@ -80,7 +80,9 @@ def NodeEvaluation(self, eval_info):
         render_image = api.Image()
         props = {}
         shader_src = "nodes/corenodes/input/noise_image_node/noise_image_node.glsl"
-        image = Image((eval_info.node.properties['width'].value, eval_info.node.properties['height'].value))
+        image_width = self.EvalProperty(eval_info, "width")
```

Comment:
Just for code readability and consistency with other nodes, maybe move the properties to the top (under render_image)